spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let(:user) { create(:user) }
+      let(:book) { create(:book) }
+      let!(:new_book_suggestion_attributes) { attributes_for(:book_suggestion, user_id: user.id) }
+      it 'creates a new book suggestion' do
+        expect do
+          post :create, params: new_book_suggestion_attributes
+        end.to change { BookSuggestion.count }.by(1)
+      end
+
+      it 'responds with 201 status' do
+        post :create, params: new_book_suggestion_attributes
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When creating an invalid book suggestion' do
+      let!(:new_book_suggestion_attributes) { attributes_for(:book_suggestion, title: nil) }
+      before do
+        post :create, params: new_book_suggestion_attributes
+      end
+
+      it 'doesn\'t create a new book suggestion' do
+        expect do
+          post :create, params: new_book_suggestion_attributes
+        end.to change { Rent.count }.by(0)
```

Comment:
`expect{ post :create, params: new_book_suggestion_attributes }.to_not change{Rent.count}`
should work too